.env.sample

```diff
@@ -2,6 +2,9 @@ PORT=3000
 
 RENDER_EXTERNAL_URL='https://framed.news'
 
+LOW_BATTERY_WARNING=10
```

Comment:
I think the LOW_BATTERY_WARNING should be per device ( see db.js > devices) and not a global default for all devices?

Reply:
On second thought, I am okay with this. Merging ...